src/main/java/calculator/Application.java

```diff
@@ -1,7 +1,20 @@
 package calculator;
 
+import calculator.controller.CalculatorController;
+import calculator.model.DelimiterExtractor;
+import calculator.model.StringCalculator;
+import calculator.view.CalculatorView;
+
 public class Application {
     public static void main(String[] args) {
+
+        DelimiterExtractor delimiterExtractor = new DelimiterExtractor();
+        StringCalculator calculator = new StringCalculator(delimiterExtractor);
```

Comment:
```suggestion
        StringCalculator calculator = new StringCalculator(new DelimiterExtractor());
```

한 줄로 함축하는 방식도 괜찮을 것 같습니다!

Reply:
아 넵! 조언 감사합니다